flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -1827,18 +1827,8 @@ public static boolean isHiveTable(Map<String, String> properties) {
 
     @Internal
     public static boolean isHiveTable(Table table) {
-        boolean isHiveTable;
-        if (table.getParameters().containsKey(CatalogPropertiesUtil.IS_GENERIC)) {
```

Comment:
I'm curious why we need to modify some logic related to Hive tables. It seems unrelated to the removal of CatalogFunction#isGeneric.